bin/build-css.sh

```diff
@@ -6,6 +6,7 @@
 # Build the application stylesheet
 ./node_modules/.bin/sass $@ \
      --load-path=. \
+     --load-path=node_modules \
```

Comment:
required to fix css for DPR - this duplicates what the template project now does